e2e/framework/utils/mysql_helper.go

```diff
@@ -0,0 +1,114 @@
+package utils
+
+import (
+	"database/sql"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/icrowley/fake"
+	"math/rand"
+	"strconv"
+	"time"
+)
+
+type MySqlHelper struct {
+	DB *sql.DB
+}
+
+// create a s3 client for specfied endpoint and creds
+func CreateNewMySqlHelper(username string, password string, url string, dbname string) *MySqlHelper {
+	dataSourceName := username + ":" + password + "@tcp(" + url + ")/" + dbname
+	db, err := sql.Open("mysql", dataSourceName)
+	if err != nil {
+		panic(err)
+	}
+
+	return &MySqlHelper{DB: db}
+}
+
+func (h *MySqlHelper) PingSuccess() bool {
+	inc := 0
+
+	for inc < 10 {
+		err := h.DB.Ping()
+
+		if err == nil {
+			return true
+		}
+		inc++
+		time.Sleep(1 * time.Second)
+	}
+
+	return false
+}
+
+//func create sample Table
+func (h *MySqlHelper) CreateTable() sql.Result {
+	result, err := h.DB.Exec("CREATE TABLE LONGHAUL (id int NOT NULL AUTO_INCREMENT,number int,data varchar(32)" +
+		", PRIMARY KEY (id) )")
+	if err != nil {
+		panic(err)
+	}
+	return result
+}
+
+//Insert Data
+func (h *MySqlHelper) InsertRandomData() sql.Result {
+	stmtIns, err := h.DB.Prepare("INSERT INTO LONGHAUL (number, data) VALUES ( ?, ? )") // ? = placeholder
+	if err != nil {
+		panic(err.Error()) // proper error handling instead of panic in your app
```

Comment:
must be copy/paste from the mysql example on go site. I will make then consistent